### ISSUE TITLE
[Bug]: Load timeout when replica num is 2 for master branch

### ISSUE THREAD

**Issue by zhuwenxing** [milvus-io/milvus#19373]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version: master-20220922-71d4a32c
- Deployment mode(standalone or cluster): cluster
- SDK version(e.g. pymilvus v2.0.0rc2): pymilvus==2.2.0.dev30
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```

### Current Behavior

collection name: deploy_test_index_type_HNSW_is_compacted_is_compacted_segment_status_all_is_vector_indexed_is_vector_indexed_is_string_indexed_is_string_indexed_replica_number_2_is_deleted_is_deleted_data_size_3000

```
[2022-09-22 09:14:56 - ERROR - pymilvus.decorators]: RPC error: [load_collection], <MilvusException: (code=-1, message=wait for loading collection timeout)>, <Time:{'RPC start': '2022-09-22 09:14:33.860359', 'RPC error': '2022-09-22 09:14:56.903346'}> (decorators.py:100)
[2022-09-22 09:14:56 - ERROR - ci_test]: Traceback (most recent call last):
  File "/home/jenkins/agent/workspace/tests/python_client/utils/api_request.py", line 26, in inner_wrapper
    res = func(*args, **_kwargs)
  File "/home/jenkins/agent/workspace/tests/python_client/utils/api_request.py", line 57, in api_request
    return func(*arg, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/pymilvus/orm/collection.py", line 474, in load
    conn.load_collection(self._name, replica_number=replica_number, timeout=timeout, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/pymilvus/decorators.py", line 101, in handler
    raise e
  File "/usr/local/lib/python3.7/dist-packages/pymilvus/decorators.py", line 97, in handler
    return func(*args, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/pymilvus/decorators.py", line 127, in handler
    ret = func(self, *args, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/pymilvus/decorators.py", line 79, in handler
    raise e
  File "/usr/local/lib/python3.7/dist-packages/pymilvus/decorators.py", line 50, in handler
    return func(self, *args, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/pymilvus/client/grpc_handler.py", line 664, in load_collection
    self.wait_for_loading_collection(collection_name, timeout)
  File "/usr/local/lib/python3.7/dist-packages/pymilvus/decorators.py", line 101, in handler
    raise e
  File "/usr/local/lib/python3.7/dist-packages/pymilvus/decorators.py", line 97, in handler
    return func(*args, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/pymilvus/decorators.py", line 127, in handler
    ret = func(self, *args, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/pymilvus/decorators.py", line 79, in handler
    raise e
  File "/usr/local/lib/python3.7/dist-packages/pymilvus/decorators.py", line 50, in handler
    return func(self, *args, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/pymilvus/client/grpc_handler.py", line 673, in wait_for_loading_collection
    return self._wait_for_loading_collection(collection_name, timeout)
  File "/usr/local/lib/python3.7/dist-packages/pymilvus/client/grpc_handler.py", line 707, in _wait_for_loading_collection
    raise MilvusException(-1, "wait for loading collection timeout")
pymilvus.exceptions.MilvusException: <MilvusException: (code=-1, message=wait for loading collection timeout)>
 (api_request.py:39)
[2022-09-22 09:14:56 - ERROR - ci_test]: (api_response) : <MilvusException: (code=-1, message=wait for loading collection timeout)> (api_request.py:40)
 -----------------------------Captured log teardown------------------------------ 
[2022-09-22 09:14:57 - INFO - ci_test]: *********************************** teardown *********************************** (test_action_first_deployment.py:33)
[2022-09-22 09:14:57 - INFO - ci_test]: [teardown_method] Start teardown test case test_task_all... (test_action_first_deployment.py:35)
[2022-09-22 09:14:57 - INFO - ci_test]: skip drop collection (test_action_first_deployment.py:36)
```

### Expected Behavior

all test cases passed

### Milvus Log

failed job: https://qa-jenkins.milvus.io/blue/organizations/jenkins/deploy_test/detail/deploy_test/282/pipeline

log:

[artifacts-cluster-reinstall-282-pytest-logs (1).tar.gz](https://github.com/milvus-io/milvus/files/9624539/artifacts-cluster-reinstall-282-pytest-logs.1.tar.gz)

[artifacts-cluster-reinstall-282-server-logs.tar.gz](https://github.com/milvus-io/milvus/files/9624540/artifacts-cluster-reinstall-282-server-logs.tar.gz)

**Comment by congqixia** [comment]
Load failure was caused by segment allocation messed among replicas. @yah01  is fixing it.

**Comment by yah01** [comment]
/assign @zhuwenxing 
fixed with #19392 , plz help check it

**Comment by zhuwenxing** [comment]
Verified and passed with version `master-20220923-6d6e14e6`
